dotCMS/src/main/webapp/html/js/dotcms/dijit/FileBrowserDialog.js

```diff
@@ -249,6 +250,7 @@ dojo.declare("dotcms.dijit.FileBrowserDialog", [dijit._Widget, dijit._Templated]
     },
 
 	_selectFolder: function(item) {
+		console.log('item', item);
```

Comment:
is needed?

Reply:
remove this

Reply:
remove https://github.com/dotCMS/core/pull/21468/commits/d5ec147fbd8c311f772b41cff27aff049bd93836